packages/shared/src/lib/core/utils/crypto/utils/validateBech32Address.ts

```diff
@@ -20,11 +18,12 @@ export function validateBech32Address(prefix: string, addr: string, addressType?
         throw new Error(localize('error.send.wrongAddressFormat'))
     }
 
+    const addressTypeLetter = addressType === undefined ? undefined : ADDRESS_TYPE_MAP[addressType] ?? undefined
     if (addressTypeLetter && !new RegExp(`^${prefix}1${addressTypeLetter}[02-9ac-hj-np-z]{58}$`).test(addr)) {
         throw new Error(localize('error.address.wrongAddressType'))
     }
 
-    let isValid = false
+    let isValid
     try {
         const decoded = Bech32.decode(addr)
         isValid = decoded && decoded.humanReadablePart === prefix
```

Comment:
I prefer this, such that `isValid` is a boolean
```suggestion
    let isValid = false
    try {
        const decoded = Bech32.decode(addr)
        isValid = !!decoded && decoded.humanReadablePart === prefix
```